component/src/test/java/com/celements/blog/metatag/HeaderSocialMediaMetaTagsTest.java

```diff
@@ -37,8 +37,8 @@ public void testIsBlogArticle_true() throws Exception {
     DocumentReference docRef = new DocumentReference(getContext().getDatabase(), "Space", "Doc");
     XWikiDocument doc = new XWikiDocument(docRef);
     getContext().setDoc(doc);
-    expect(blogServiceMock.getBlogConfigDocRef(same(
-        doc.getDocumentReference().getLastSpaceReference()))).andReturn(docRef);
+    expect(blogServiceMock.getBlogConfigDocRef(doc.getDocumentReference().getLastSpaceReference()))
```

Comment:
why changing argument matcher from `same` to `equals` (default)?

Reply:
because the spaceReference isn't the same instance anymore, but a clone.
`same` is too strict, `equals` suffices.